__snapshots__/test.js.snap

```diff
@@ -1,5 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`custom chunks 1`] = `
```

Comment:
How can we know from this snapshot that it works correctly?

Reply:
Without the `chunks` filter, it will generate a `script` tag against all entries. I'll have a look at the test, though, to see if we can simplify or state this better.